### ISSUE TITLE
DITA 1.3 Subject Scheme Map XML Schema Based considerd invalid

### ISSUE THREAD

**Issue by raducoravu** [dita-community/org.oasis-open.dita.dita13.doctypes#2]
When I validate with OxygenXML specific DITA validation:

```
        <subjectScheme xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="urn:oasis:names:tc:dita:spec:classification:xsd:subjectScheme.xsd:1.3">
            <subjectHead>
                <subjectHeadMeta>
                    <navtitle>
                        <!-- Enter a description of the subject scheme. -->
                    </navtitle>
                </subjectHeadMeta>
            </subjectHead>
        </subjectScheme>
```

I obtain the error:

```
       The "domains" attribute is missing from the element subjectScheme. Probably the DITA specialization is not correctly constructed.
```

**Comment by drmacro** [comment]
Package had bad copy of the XSD. Corrected by fix to issue #1

**Comment by raducoravu** [comment]
I checked and both issues are fixed, thanks.
